tinodesdk/src/main/java/co/tinode/tinodesdk/Tinode.java

```diff
@@ -1372,6 +1373,22 @@ public void onFinally() {
         return sendWithPromise(msg, msg.leave.id);
     }
 
+    /**
+     * Headers to be sent with an outgoing message.
+     *
+     * @param content message content
+     * @return headers in as map "header key : header value"
+     */
+    public static Map<String, Object> getDraftyHeaders(final Drafty content) {
```

Comment:
I would rename the method to something like `draftyHeadersFor` or `createHeadersFor` or `newHeadersFrom`. The `getX` implies that the value of X is extracted from the target. In this case it's not extracted. It's being created.

Reply:
Agreed. Thanks for suggestion.